StockPriceScraper/Program.cs

```diff
@@ -52,7 +52,7 @@ public static async Task Main(string[] args)
             
             Console.WriteLine("Starting downloads.");
             
-            await RunDownloadStockDataAsync();
+            await RunDownloadStockDataAsync().ConfigureAwait(false);
```

Comment:
why do u use the additional method 'ConfigureAwait(false)'. Is the default value 'true'?